habitat-baselines/habitat_baselines/rl/ddppo/transforms.py

```diff
@@ -0,0 +1,104 @@
+from typing import Optional
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms.functional as TF
+from torchvision.transforms import ColorJitter, RandomApply
+
+
+class RandomShiftsAug(nn.Module):
```

Comment:
class description - when should i use this? can be simple

Reply:
done